sdk/communication/communication-chat/CHANGELOG.md

```diff
@@ -1,10 +1,13 @@
 # Release History
 
-## 1.0.0-beta.5 (Unreleased)
+## 1.0.0-beta.5 (2021-03-09)
 
 ### Breaking Changes
 
 - Removed `_response` from returned models.
+- Updated to @azure/communication-common@1.0.0-beta.6. Now uses `CommunicationIdentifier` in place of `CommunicationUserIdentifier`.
+- Swap the param order in `ChatThreadClient` constructer.
```

Comment:
Swap the parameter order in `ChatThreadClient` constructor.

Reply:
and typo nit: constructor with an o